internal/app/machined/internal/phase/rootfs/etc/etc.go

```diff
@@ -127,7 +64,7 @@ func OSRelease() (err error) {
 
 	tmpl, err := template.New("").Parse(osReleaseTemplate)
 	if err != nil {
-		return
+		return err
```

Comment:
that's where it becomes confusing imho: if we have function result declared as `(err error)`, what does `tmpl, err := ` do ? I believe it creates another `err` which shadows previous `err`, so bare `return` returns wrong `err`

Reply:
should be easy enough to change. i just did a blind copy paste of that bit, whoops.